recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -1,10 +1,5 @@
 FILESEXTRAPATHS:prepend := "${THISDIR}/${PN}:"
```

Comment:
@aniket-l We can remove this line!

Reply:
@cshilwant oops, I merged the PR before seeing the update

Reply:
@aniket-l Ohh, no issues. Can you handle it in a separate PR?

Reply:
@cshilwant Yeah sure will do!